torch/utils/_python_dispatch.py

```diff
@@ -391,6 +391,11 @@ def is_traceable_wrapper_subclass(t: object) -> TypeGuard[TensorWithFlatten]:
         and hasattr(t, "__tensor_unflatten__")
     )
 
+def is_traceable_wrapper_subclass_type(t: Type) -> TypeGuard[Type[TensorWithFlatten]]:
```

Comment:
output of this fn should be `-> bool:` right?

Reply:
(oh same with the one above but I guess that is old)

Reply:
also nit: can you just update `is_traceable_wrapper_subclass` to `return is_traceable_wrapper_subclass_type(type(x))` to dedup the code?

Reply:
Oh I thought TypeGuard is just a special bool, right?